File: browser/src/model/post.ts
export interface PostItem {
  id:number; //
  did:number; // 所属部门
  code:string; // 岗位编码
  name:string; // 岗位名称
  status:string; // 状态
  recruit:number;//招聘人数
  createBy:number; // 创建者
  createTime:Date; // 创建时间
  updateBy:number; // 更改者
  updateTimeL:Date; // 更改时间
}

File: Server/src/main/java/cn/edu/zqu/hr_system/project/model/entities/Post.java
package cn.edu.zqu.hr_system.project.model.entities;

import cn.edu.zqu.hr_system.project.base.BaseEntity;
import lombok.Data;

@Data
public class Post extends BaseEntity {
    Long did; // 所属部门
    String code; // 岗位编码
    String name; // 岗位名称
    Character status; // 状态
    Integer recruit;//招聘人数
}

